packages/postgres-database/src/simcore_postgres_database/models/resource_tracker.py

```diff
@@ -73,8 +75,63 @@
         sa.DateTime(timezone=True),
         nullable=False,
         doc="Last prometheus scraped timestamp (UTC timestamp)",
+        index=True,
     ),
     column_modified_datetime(timezone=True),
+    sa.Column(
+        "node_uuid",
+        sa.String,
+        nullable=False,
+        doc="node_uuid label scraped via Prometheus (taken from container labels)",
+    ),
+    sa.Column(
+        "node_label",
+        sa.String,
+        nullable=True,
+        doc="node label",
+    ),
+    sa.Column(
+        "instance",
+        sa.String,
+        nullable=True,
+        doc="instance label scraped via Prometheus (taken from container labels)",
+    ),
+    sa.Column(
+        "service_settings_limit_nano_cpus",
+        sa.BigInteger,
+        nullable=True,
+        doc="CPU resource limit allocated to a container, ex.500000000 means that the container has limit for 0.5 CPU shares",
+    ),
+    sa.Column(
+        "service_settings_limit_memory_bytes",
+        sa.BigInteger,
+        nullable=True,
+        doc="memory limit in bytes scraped via Prometheus",
+    ),
+    sa.Column(
+        "project_name",
+        sa.String,
+        nullable=True,
+        doc="project name",
+    ),
+    sa.Column(
+        "user_email",
+        sa.String,
+        nullable=True,
+        doc="user email",
```

Comment:
I guess you agree with me that this doc  (and may of the others above) is redundant and not very informative. I would rather explain here things as:
- why these entries are not foreign keys
- what is the purpose of storing this information (i.e. what you explained to me offline about dumping data and decoupling from the other tables, etc)